### PR TITLE
Changes in creating new committee structure

### PR DESCRIPTION
**Pull request by RaghavPro**
#149
I created two form subclasses for creating and editing committee structure.
Edit form has a select list for selecting place level and Add form has an input field.

### CODE REVIEW

File: cleansweep/plugins/committees/forms.py [hunk 1]
```diff
@@ -53,6 +51,29 @@ def ensure_empty_slots(self, n=5):
...
+    """
+    Edit form for editing committee structures. Extends CommitteeForm.
+    """
+    level = SelectField('Level', choices=[])  # Select field for place level selection.
+
+    def __init__(self, place, *a, **kw):
+        super(EditCommitteeForm, self).__init__(place, *a, **kw)
+        place_types = [place.type] + place.type.get_subtypes()
+        self.level.choices = [(t.short_name, t.name) for t in place_types]
+
```
Comment (anandology) [inline cleansweep/plugins/committees/forms.py]: Do we really need two different forms here (three if you count the base class)? Why not get rid of the level dropdown completely. For new committees, the level is already available from the URL and for the existing committees, we don't allow changing that.

Comment (RaghavPro) [inline cleansweep/plugins/committees/forms.py]: If you're sure that we won't be changing levels in committee structures then we don't.
In #122 you [said](https://github.com/AamAadmiParty/cleansweep/issues/122#issuecomment-142810296) it should be allowed only if there's no other structure at that level.

Comment (anandology) [inline cleansweep/plugins/committees/forms.py]: That is not valid any more. You can assume that we are not going to allow moving a committee-structure to another level. 

We are slowly moving towards a site-wide admin panel where these things are defined initially and rarely modified.

Comment (RaghavPro) [inline cleansweep/plugins/committees/forms.py]: Oh, all right. I should've asked before changing. I'll undo it.

**Comment by anandology** [comment]
Thanks @RaghavPro. This looks great.

**Pull request merged**